scripts/generateBandageLabelsFromAlignment.py

```diff
@@ -0,0 +1,117 @@
+from collections import defaultdict
```

Comment:
According to [this](https://docs.python.org/3/library/collections.html), the `collections` module is deprecated and will be removed in Python 3.10 (and we are at 3.8). Consider using the replacement instead, `collections.abc`.

Reply:
I tried to find some more information about this, and I believe the deprecation warning does not actually apply to `defaultdict`:
https://stackoverflow.com/questions/60440157/is-there-a-replacement-for-python-namedtuple

If it did, I believe Python would have given me a warning during run time. 

You can see from [here](https://docs.python.org/3.5/library/collections.abc.html#collections-abstract-base-classes) that there used to be an abc submodule that belonged to `collections`:

> New in version 3.3: Formerly, this module was part of the collections module.

Reply:
Admittedly very confusing though